golos.charge/golos.charge.cpp

```diff
@@ -33,13 +34,15 @@ fixp_t charge::consume_charge(name issuer, name user, symbol_code token_code, ui
                 item.charge_id = charge_id;
                 item.last_update = current_time();
                 item.value = new_val.data();
+                send_charge_event(user, item);
             });
         return new_val;
     }
     if (new_val > 0)
         balances_table.modify(*itr, name(), [&]( auto &item ) {
             item.last_update = current_time();
             item.value = new_val.data();
+            send_charge_event(user, item);
         });
     else
         balances_table.erase(itr);
```

Comment:
now here is `send_event` on `emplace` and `modify`, but no notification on `erase`